lang/LangPrimSource/SC_ComPort.h

```diff
@@ -20,103 +20,122 @@
 
 #pragma once
 
+#include "SC_ReplyImpl.hpp"
 #include "SC_Types.h"
 
 #include <boost/array.hpp>
 #include <boost/asio.hpp>
 #include <boost/enable_shared_from_this.hpp>
+#include <boost/core/noncopyable.hpp>
 
 //////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 const int kTextBufSize = 65536;
 
-class SC_UdpInPort {
-    int mPortNum;
-    boost::array<char, kTextBufSize> recvBuffer;
+enum class HandlerType { OSC, Raw };
 
-    boost::asio::ip::udp::endpoint remoteEndpoint;
+using HandleDataFunc = std::function<void(std::unique_ptr<char[]>, size_t)>;
```

Comment:
In this case, we are actually handing off ownership of this data, so std::span doesn't capture it? This could be re-worked as `std::string`, but probably that's more of a refactoring change after this PR.